docs/scos/dev/back-end-development/zed-ui-tables/create-and-configure-zed-tables.md

```diff
@@ -131,23 +131,39 @@ $config->setDefaultSortDirection(
 
 ### Configure strict search
```

Comment:
```suggestion
### Configure search by columns
```

Please update the documentation below, as we would have just added a search by columns, not a `strict search`.
Please make sure, that this paragraph only refers to the existing search for comparison reasons.